source/codegen/templates/proto_helpers.mako

```diff
@@ -69,66 +69,79 @@ ${lookup.get_template(custom_template).render()}
 </%def>
 
 ## Define a proto request message for a given API function.
-<%def name="define_request_message(function, input_parameters)">\
+<%def name="define_request_message(function, input_parameters, proto_values)">\
```

Comment:
Note: Changes here and in `define_response_message` below are for consuming proto-value overrides provided through metadata.